content/en/serverless/installation/ruby.md

```diff
@@ -18,7 +18,7 @@ aliases:
     - /serverless/datadog_lambda_library/ruby/
 ---
 
-<div class="alert alert-warning">If you previously set up your Lambda functions using the Datadog Forwarder, see <a href="https://docs.datadoghq.com/serverless/guide/datadog_forwarder_ruby">instrumenting using the Datadog Forwarder</a>. Otherwise, follow the instructions in this guide to instrument using the Datadog Lambda Extension.</div>
+<div class="alert alert-warning">If you previously set up your Lambda functions using the Datadog Forwarder or plan on using custom metrics in your lambda fucntions, see <a href="https://docs.datadoghq.com/serverless/guide/datadog_forwarder_ruby">instrumenting using the Datadog Forwarder</a>. Otherwise, follow the instructions in this guide to instrument using the Datadog Lambda Extension.</div>
```

Comment:
```suggestion
<div class="alert alert-warning">If you previously set up your Lambda functions using the Datadog Forwarder or plan on using custom metrics in your Lambda functions, see <a href="https://docs.datadoghq.com/serverless/guide/datadog_forwarder_ruby">instrumenting using the Datadog Forwarder</a>. Otherwise, follow the instructions in this guide to instrument using the Datadog Lambda Extension.</div>
```